pom.xml

```diff
@@ -41,6 +41,7 @@
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<junit.platform.version>1.3.1</junit.platform.version>
 		<junit.jupiter.version>5.3.1</junit.jupiter.version>
+		<osgi.version>4.3.1</osgi.version>
```

Comment:
Stable version 5.0.0 is available for both dependencies. Let's update.

Reply:
We should not require a 5.0.0 Version so it is possible to install the driver in older frameworks, since this is only API there is no such thing like 'bugfixing' between 4.x and 5x